app/boards/arm/preonic/preonic_rev3.dts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <st/f3/stm32f303Xc.dtsi>
+#include <dt-bindings/zmk/matrix-transform.h>
+
+/ {
+	model = "Preonic PCD, rev3";
+	compatible = "preonic,rev3", "st,stm32f303";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zmk,kscan = &kscan0;
+        zmk,matrix_transform = &layout_grid_transform;
```

Comment:
Some indenting inconsistency here.

Reply:
Corrected all the formatting issues in this file.